CONTRIBUTING.md

```diff
@@ -2,32 +2,61 @@
 Welcome! Thanks for checking out Expensify.cash and for taking the time to contribute!
 
 ## Getting Started
-This guide is specifically for external contributors. For a general overview of the repo, check out our README located [here](https://github.com/Expensify/Expensify.cash/blob/master/README.md). The part of the README to pay particular attention to is how to [run the app](https://github.com/Expensify/Expensify.cash#local-development) locally using our production API.
+If you would like to become an Expensify.cash contributor, the first step is to read this document in it's entirety. The second step is to review the README guidelines [here](https://github.com/Expensify/Expensify.cash/blob/master/README.md) for a general overview of the code repository (i.e. how to run the app locally, testing, storage, etc). Please read both documents before asking questions, as it may be covered within the documentation.
 
 #### Test Accounts
-You can create as many accounts as needed in order to test your changes directly from [expensify.cash](https://expensify.cash/). An initial account can be created when logging in for the first time and additional accounts can be invited by entering a valid email or phone in the "Find or start a chat" input then tapping the avatar.
+You can create as many accounts as needed in order to test your changes directly from [expensify.cash](https://expensify.cash/). An initial account can be created when logging in for the first time, and additional accounts can be invited by entering a valid email or phone in the "Find or start a chat" input then tapping the avatar.
 
 ## Code of Conduct
 This project and everyone participating in it is governed by the Expensify [Code of Conduct](https://github.com/Expensify/Expensify.cash/blob/master/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [contributors@expensify.com](mailto:contributors@expensify.com).
 
 ## Asking Questions
-The best way to ask questions is to join our #expensify-contributors Slack channel. To request an invite to the channel, just email contributors@expensify.com with the subject "Slack Channel Invite" and we'll send you an invite! Please do not create issues to ask questions.
+If you have any general questions, please ask in the #expensify-contributors Slack channel. To request an invite to the channel, just email contributors@expensify.com with the subject `Slack Channel Invite` and we'll send you an invite! The Expensify team will not be able to respond to direct messages in Slack.
```

Comment:
I think we should still clarify that GH issues shouldn't be created for the sole purpose of asking questions.

Reply:
Hm I don't agree because I think we start opening the path to listing all ways to *not ask a question* if we do this type of approach. Right now I think it's really clear how/when to ask questions, and if we see someone violate that rule we just treat it like our manifesto (point them to the Contributing.md or Readme.md files, and ask them to re-read it).

I feel like listing it out is too exhaustive and specific - like we wouldn't want to get caught up saying "don't ask questions by... making new GH issues, messaging us in Upwork, emailing contributors@expensify.com, posting on the community, reaching out to concierge, etc"

Reply:
Can we try out this suggestion, and circle back to update this copy if it's still generating issues to make the update you are proposing? I'm really hoping to keep brevity here so people actually read this guide 🤞 (but hey, time will tell if these edits actually help lol)

Reply:
Yeah, that works! We originally included it there since it is common for GH issues to be used to ask questions in open-source repositories, but I agree we can always re-add it if it actually becomes a problem

Reply:
>since it is common for GH issues to be used to ask questions in open-source repositories

This argument is the first one I'll remember if contributors continue to do this in the future (I've never been an external contributor of another project so I didn't know this!). Let me know if our contributors keep doing this.

Reply:
Probably fine to take out, but I agree that we don't want people creating GH issues to ask questions.